### PR TITLE
Issue #37

### PR DESCRIPTION
**Pull request by XyDrKRulof**

### CODE REVIEW

File: gym_electric_motor/physical_systems/voltage_supplies.py [hunk 1]
```diff
@@ -62,3 +66,49 @@ def __init__(self, u_nominal=600.0, **__):
     def get_voltage(self, *_, **__):
         # Docstring of superclass
         return self._u_nominal
+
+
+class RCVoltageSupply(VoltageSupply):
+    """Voltage supply moduled as RC element"""
+    
+    def __init__(self, u_nominal=600.0, supply_parameter={'R':1,'C':4e-3}, **__):
```
Comment (atra94) [inline gym_electric_motor/physical_systems/voltage_supplies.py:74]: Mutable variables (the dictionary) should not be used as default parameters in function headers. (See: http://google.github.io/styleguide/pyguide.html Section: 2.12.4 Decision)

Instead use: 
`supply_parameter=None` as default value
and 
`supply_parameter=supply_parameter or {'R': 1, 'C': 4e-3}`
inside the function

File: gym_electric_motor/physical_systems/voltage_supplies.py [hunk 2]
```diff
@@ -62,3 +66,49 @@ def __init__(self, u_nominal=600.0, **__):
     def get_voltage(self, *_, **__):
         # Docstring of superclass
         return self._u_nominal
+
+
+class RCVoltageSupply(VoltageSupply):
+    """Voltage supply moduled as RC element"""
```
Comment (atra94) [inline gym_electric_motor/physical_systems/voltage_supplies.py:72]: I think you meant  'modeled', right?

File: gym_electric_motor/physical_systems/voltage_supplies.py [hunk 3]
```diff
@@ -62,3 +66,49 @@ def __init__(self, u_nominal=600.0, **__):
...
+
+
+class RCVoltageSupply(VoltageSupply):
+    """Voltage supply moduled as RC element"""
+    
+    def __init__(self, u_nominal=600.0, supply_parameter={'R':1,'C':4e-3}, **__):
+        """
+        This Voltage Supply is a model of a non ideal Voltage supply where the ideal voltage source U_0 is part of an RC element
+        Args:
+            supply_parameter(dict): Consists or Reluctance R in Ohm and Capacitance C in Farad
```
Comment (atra94) [inline gym_electric_motor/physical_systems/voltage_supplies.py:78]: R is the resistance not a reluctance

**Comment by XyDrKRulof** [comment]
Pushed all requested changes.

**Pull request merged**